content/realtime/channels.textile

```diff
@@ -540,6 +575,13 @@ channel.subscribe("action") { message in
 }
 channel.publish("action", data: "boom!")
 
+bc[flutter]. final channel = realtime.channels.get('chatroom');
+/* Implicit attach when subscribing */
+channel.subscribe(name: 'action').listen((ably.Message message){
+    print('Received: ${message.data}');
```

Comment:
Inconsistent indentation and use of `{` without a space above.

Reply:
addressed in e288898cb0f972b74350bd69c2da054346fbb4f0